### PR TITLE
Bug 1442223, clarified details around Docker storage configuration

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1442223

### CODE REVIEW

File: install_config/install/host_preparation.adoc [hunk 1]
```diff
@@ -293,10 +293,20 @@ accordingly.
...
 
-Containers and the images they are created from are stored in Docker's
-storage back end. This storage is ephemeral and separate from any
-xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[persistent storage] allocated to
-meet the needs of your applications.
+Containers and the images they are created from are stored in Docker's storage
+back end. This storage is ephemeral and separate from any
+xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[persistent
+storage] allocated to meet the needs of your applications. With _Ephemeral
+storage_, container-saved data will be lost once the container is removed. With
```
Comment (kalexand-rh) [inline install_config/install/host_preparation.adoc]: s/will be lost once/is lost when

File: install_config/install/host_preparation.adoc [hunk 2]
```diff
@@ -293,10 +293,20 @@ accordingly.
...
+_persistent storage_, container-saved data remains if the container is removed.
+
+Every system running a container daemon needs container storage configured. For
```
Comment (kalexand-rh) [inline install_config/install/host_preparation.adoc]: maybe s/Every system running a container daemon needs container storage configured./You must configure storage for each system that runs a container daemon.

File: install_config/install/host_preparation.adoc [hunk 3]
```diff
@@ -293,10 +293,20 @@ accordingly.
...
+containerized installations, you need storage on masters. Also, by default, the
+web console is run in containers on masters and storage is needed on masters to
```
Comment (kalexand-rh) [inline install_config/install/host_preparation.adoc]: s/masters/masters,

File: install_config/install/prerequisites.adoc [hunk 4]
```diff
@@ -89,9 +89,9 @@ endif::[]
 * Minimum 15 GB hard disk space for the file system containing *_/var/_*. image:redcircle-1.png[]
 * Minimum 1 GB hard disk space for the file system containing *_/usr/local/bin/_*.
 * Minimum 1 GB hard disk space for the file system containing the system's temporary directory. image:redcircle-2.png[]
-* An additional minimum 15 GB unallocated space to be used for Docker's storage
+* An additional minimum 15 GB unallocated space (per system running containers) to be used for Docker's storage
```
Comment (kalexand-rh) [inline install_config/install/prerequisites.adoc]: s/(per system running containers) to be used for/per system that runs containers for

File: install_config/install/prerequisites.adoc [hunk 5]
```diff
@@ -89,9 +89,9 @@ endif::[]
...
 back end; see xref:../../install_config/install/host_preparation.adoc#configuring-docker-storage[Configuring
-Docker Storage].
+Docker Storage]. Additional space may be required, depending on the size and number of containers being run on the node.
```
Comment (kalexand-rh) [inline install_config/install/prerequisites.adoc]: s/may/might
s/being run/that run

File: install_config/install/prerequisites.adoc [hunk 6]
```diff
@@ -121,6 +121,14 @@ module in Python's standard library.
 {product-title} only supports servers with x86_64 architecture.
 ====
 
+Every system running a container daemon needs container storage configured. For
```
Comment (kalexand-rh) [inline install_config/install/prerequisites.adoc]: same suggestions as above

**Review by kalexand-rh — COMMENTED** [review]
This sounds really helpful! I have a couple of picks.

**Comment by ahardin-rh** [comment]
@kalexand-rh Excellent suggestions. Thank you! 

QE approved in the BZ.

**Pull request merged**